addOns/exim/src/main/java/org/zaproxy/addon/exim/log/LogsImporter.java

```diff
@@ -112,7 +116,8 @@ private static void processModSecLogs(ModSecurity2AuditReader reader) throws IOE
                 }
                 break;
             } catch (Exception e) {
-                LOG.error(e.getMessage(), e);
+                LOG.warn(e.getMessage());
```

Comment:
Passing the exception (e) displays the whole stack dump, it no users should need to see that (for my two cents).

Reply:
To be clear, I changed from error to warn. In the warn only show the message, in debug show the stack trace. (In case we need it for debugging in the future.)

Reply:
It does not have that effect though, calling just with the exception will log just the `toString()` result.

Reply:
I’ll test further.

Reply:
Okay I see, when it's the only item then it toString's it. I think the warn will be fine, I'm going to dump the debugs. If we have to chase something down later we can always do something temporary.